app/src/docs/v1.api-spec.yaml

```diff
@@ -3,33 +3,1073 @@ openapi: 3.0.2
+      - in: path
+        name: submissionId
+        schema:
+          type: string
+          format: uuid
+        required: true
+      requestBody:
+        description: The form's data sent when the form is edited by a user
+        required: true
+        content:
+          application/json:
+            schema:
+              type: array
+              items:
+                type: object
+                $ref: '#/components/schemas/SubmissionPost'
+      responses:
+        '201':
+          description: Submission updated
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Submission'
+    delete:
+      tags:
+      - Submissions
+      parameters:
+        - in: path
+          name: slug
+          schema:
+            type: string
+          required: true
+          description: a short name for the Form
+        - in: path
+          name: submissionId
+          schema:
+            type: string
+            format: uuid
+          required: true
+      summary: Soft-deletes the submission with the specified ID.
+      responses:
+        '200':
+          description: Submission was deleted.
+
+
+  /{slug}/submissions/{submissionId}/pdf:
+    get:
+      summary: Generate a form submission as a PDF document
+      operationId: generateSubmissionPdf
+      tags:
+        - Submissions
+      parameters:
+        - in: path
+          name: slug
+          schema:
+            type: string
+          required: true
+          description: a short name for the Form
+        - in: path
+          name: submissionId
+          schema:
+            type: string
+            format: uuid
+          required: true
+      responses:
+        '201':
+          description: PDF file generated with the CDOGS API
+          content:
+            application/pdf:
+              schema:
+                type: string
+                format: binary
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        default:
+          description: Unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+
+
+
+  /{slug}/submissions/email:
+    post:
+      summary: Send Email notification to Admin
+      operationId: sendSubmissionEmail
+      tags:
+        - Submissions
+      parameters:
+        - in: path
+          name: slug
+          schema:
+            type: string
+          required: true
+          description: a short name for the Form
+      requestBody:
+        description: A Submission object
+        required: true
+        content:
+          application/json:
+            schema:
+              type: array
+              items:
+                type: object
+                $ref: '#/components/schemas/Submission'
+      responses:
+        '200':
+          description: https://ches-master-9f0fbe-prod.pathfinder.gov.bc.ca/api/v1/docs#operation/postMerge
```

Comment:
I don't think this is good practice to cross reference like that. There may be a better way to cross-reference the response object style but not sure off the top of my head.

Reply:
i suspect a CORS exception needs to be configured on the CHES server to allow us to link directly to that schema.
I've put a link in the request decription instead.